### PR TITLE
Inject by annotation

### PR DESCRIPTION
**Pull request by alexwatcher**
Some Refactoring, use annotation 'integrity-monitor/inject' to apply injection patch

### CODE REVIEW

**Review by Denislite — APPROVED** [review]
LGTM

File: cmd/main.go [hunk 1]
```diff
@@ -70,5 +24,8 @@ func main() {
 		MaxHeaderBytes: 1 << 20, // 1048576
 	}
 
-	log.Fatal(s.ListenAndServeTLS("./ssl/k8s-webhook-injector.pem", "./ssl/k8s-webhook-injector.key"))
+	logger.Info("Start Integrity Monitor Injector webhook server")
+	if err := s.ListenAndServeTLS("./ssl/k8s-webhook-injector.pem", "./ssl/k8s-webhook-injector.key"); err != nil {
```
Comment (siarhei-shliayonkin) [inline cmd/main.go:28]: Hard coded cert and key. 
TODO: get from args.

**Review by alavrovinfb — APPROVED** [review]
looks good

File: pkg/mutate/mutate.go [hunk 2]
```diff
@@ -5,73 +5,65 @@ import (
...
 	}
 
-	var patch string
-	patchType := v1.PatchTypeJSONPatch
-
-	log.Println("pod has following labels", pod.Labels)
-	if _, ok := pod.Labels["hasher-certificates-injector-sidecar"]; ok {
-		file, err := os.Open("/app/patch-json-command.json")
+	logEntry := logger.WithField("Pod", pod.Name)
+	logEntry.WithField("Annotations", pod.Annotations).Debug("Process Pod")
```
Comment (siarhei-shliayonkin) [inline pkg/mutate/mutate.go:38]: You may use logEntry.WithFields(...) to combine several fields

**Review by siarhei-shliayonkin — COMMENTED** [review]
A few minor notices.

**Pull request merged**